alyohea/ggrc-core
# GGRC-7779 Add tooltip to the GCA attributes with long title

# Issue description
Add tooltip to the GCA attributes with long title.

# Steps to test the changes 
1. Login in GGRC app. 
2. Open Administration page. 
3. Add GCA attribute with long title e.g to the Objective.  
4. Open any Objective object and review created GCA. 
**Actual: A part of the GCA's title is cut and there is no possibility to view all the title. 
**Expected: Full GCA's title is be visible in toolltip.  

# Solution description 
Add tooltip when we hover over the title. 

# Sanity checklist 

# PR Review checklist
